week-6/Homework/mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +490,37 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+document.getElementById("newqoute").addEventListener("click", changeQoute);
+let title = document.getElementById("qoute");
+let p = document.getElementById("author");
+let qouteObject = pickFromArray(quotes);
+title.innerHTML = qouteObject.quote;
+p.innerText = qouteObject.author;
+
+function changeQoute() {
+  let qouteObject = pickFromArray(quotes);
+  title.innerHTML = qouteObject.quote;
+  p.innerText = qouteObject.author;
+}
+let switchButton = document
+  .getElementById("switch")
+  .addEventListener("click", switchAuto);
+
+let intervalId = -1;
+function switchAuto() {
+  if (intervalId == -1) {
+    intervalId = setInterval(() => {
+      let qouteObject = pickFromArray(quotes);
+      title.innerHTML = qouteObject.quote;
+      p.innerText = qouteObject.author;
```

Comment:
The same three lines are repeated in multiple places.

How would you avoid repeating them?